src/libstd/io/error.rs

```diff
@@ -62,12 +62,18 @@ pub type Result<T> = result::Result<T, Error>;
 /// [`Write`]: ../io/trait.Write.html
 /// [`Seek`]: ../io/trait.Seek.html
 /// [`ErrorKind`]: enum.ErrorKind.html
-#[derive(Debug)]
 #[stable(feature = "rust1", since = "1.0.0")]
 pub struct Error {
     repr: Repr,
 }
 
+#[stable(feature = "io_error_debug", since = "1.24.0")]
```

Comment:
As  `Debug` has been implemented for `io::Error` since 1.0.0 `#[stable(feature = "rust1", since = "1.0.0")]` would make more sense here.

Reply:
Will do